lib/cog/commands/echo.ex

```diff
@@ -4,11 +4,12 @@ defmodule Cog.Commands.Echo do
   @moduledoc """
   Repeats whatever it is passed.
 
-  ## Example
-
-      @bot #{Cog.embedded_bundle}:echo this is nifty
-      > this if nifty
+  USAGE:
+    echo [args]
```

Comment:
I like using that `...` to differentiate between 1 arg and multiple. It's pretty obvious here but, it's still nice to be consistent.